src/libcore/tests/ascii.rs

```diff
@@ -143,8 +142,6 @@ macro_rules! assert_all {
                            stringify!($what), b);
                 }
             }
-            assert!($str.$what());
-            assert!($str.as_bytes().$what());
```

Comment:
These methods are removed on `str` and `[u8]` in favor of explicit `Iterator::all`: https://github.com/rust-lang/rust/issues/39658#issuecomment-333977502